examples/runners/wgpu/builder/src/main.rs

```diff
@@ -15,9 +23,13 @@ fn build_shader(path_to_create: &str, codegen_names: bool) -> Result<(), Box<dyn
 }
 
 fn main() -> Result<(), Box<dyn Error>> {
-    build_shader("../../shaders/sky-shader", true)?;
-    build_shader("../../shaders/simplest-shader", false)?;
-    build_shader("../../shaders/compute-shader", false)?;
-    build_shader("../../shaders/mouse-shader", false)?;
+    build_shader("../../shaders/sky-shader", true, &[])?;
+    build_shader("../../shaders/simplest-shader", false, &[])?;
+    build_shader("../../shaders/compute-shader", false, &[])?;
+    build_shader(
+        "../../shaders/mouse-shader",
+        false,
+        &[Capability::Int8, Capability::Int16, Capability::Int64],
```

Comment:
Maybe open an issue for this? I think it's just one of those `num-traits` things like `.abs()`?

Reply:
https://github.com/EmbarkStudios/rust-gpu/issues/634